packages/code-du-travail-api/src/server/routes/items/getRelatedItems.js

```diff
@@ -126,12 +123,11 @@ async function getSearchBasedItems({ title, settings }) {
   );
```

Comment:
one line above, instead of creating a high order function I would simply use the spread operator:

`.map(({ _source }) => mapSource("search")(_source))` => `.map(({ _source }) => ({..._source, reco: "search"})`

You are less likely to miss a field if we change some

Reply:
(but if the goal is to also filter some fields, nevermind 😄 )

Reply:
yep, I'm also selecting a subset of the fields